jekyll/_docs/install-and-use-yarn.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: classic-docs
+title: "Install and Use Yarn (the NPM replacement) on CircleCI"
+short-title: "Install and Use Yarn on CircleCI"
+categories: [how-to]
+description: "How to modify `circle.yml` in order to install and use Yarn on CircleCI."
+---
+
+<img src="{{site.baseurl}}/assets/img/logos/yarn-logo.svg" style="display:block;margin:15px auto;width:40%;min-width:320px;" alt="Yarn Logo" />
+
+[Yarn][yarn-site] is an open-source package manager for JavaScript. Both Yarn itself as well as the packages it installs can be cached on CircleCI. This can potentially speed up builds but more importantly can remove network connectivity as a source of problems.
+
+## Setup
+
+```
+machine:
+  environment:
+    YARN_VERSION: 0.17.8
+    PATH: "${PATH}:${HOME}/.yarn/bin"
+```
+
+The Yarn install script seen later can be used without a version. That will download the latest stable version of Yarn. Here a Yarn version is set in order to reduce the number of variables in a build.
```

Comment:
I don't think it makes sense to mention a script that the reader hasn't seen yet. Better might be:

"We've set the Yarn version here to reduce the number of variables in the build, but you don't need to specify a version when actually installing Yarn; the default setting is the latest stable version."